src/main/java/core/basesyntax/OperationsStrategy.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class OperationsStrategy {
+
+    public OperationsStrategy(Fruits fruits) {
+        this.fruits = fruits;
+    }
+
+    private Fruits fruits;
+
+    public int operation(String operation, String fruitType, int amount) {

```

Comment:
Consider validating the `operation` parameter before the switch statement to ensure it is not null or empty, which would prevent potential runtime errors.